src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/ElasticSearchDAO.scala

```diff
@@ -93,4 +99,20 @@ class ElasticSearchDAO(servers:Seq[Authority], indexName: String) extends Search
       case e: Exception => logger.warn(s"ES index '%s' could not be recreated and may be in an unstable state.".format(indexName))
     }
   }
+
+  override def findDocuments(term: String, from: Int = 0, size: Int = 10) : LibrarySearchResponse = {
```

Comment:
Might be nicer if findDocuments took in term as an Option(String) and then you could just match the None case to use findAll vs make the queryStr from the String